File: API/Entities/Book.cs
namespace API.Entities
{
    public class Book
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public long Price { get; set; }
        public string PictureUrl { get; set; }
        public string Author { get; set; }
        public string Genre { get; set; }
        public int YearOfRelease { get; set; }
        public int NumberOfPages { get; set; }
    }
}

File: client/src/app/models/book.ts
export interface Book {
    id: number;
    name: string;
    description: string;
    price: number;
    pictureUrl: string;
    author: string;
    genre: string;
    yearOfRelease: string;
    numberOfPages: string;
}

export interface BookParameters {
    orderBy: string;
    search?: string;
    genres: string[];
    author: string;
    pageNumber: number;
    pageSize: number;
}